pom.xml

```diff
@@ -95,7 +95,7 @@
         <version.org.codehaus.woodstox.woodstox-core>5.0.3</version.org.codehaus.woodstox.woodstox-core>
         <version.org.fusesource.jansi>1.11</version.org.fusesource.jansi>
         <version.org.jboss.aesh>0.66.11</version.org.jboss.aesh>
-        <version.org.jboss.byteman>3.0.3</version.org.jboss.byteman>
+        <version.org.jboss.byteman>4.0.0-ALPHA</version.org.jboss.byteman>
```

Comment:
I don't think byteman is problematic as it is only test dependency and as such should be fine to use in our build / testsuite.
Non tests dependencies are different story.